app/src/components/dialog/EditDialog.tsx

```diff
@@ -23,7 +23,7 @@ export interface IEditDialogProps<T> {
    * @type {string}
    * @memberof IEditDialogProps
    */
-  dialogTitle: string;
+  dialogTitle: string | JSX.Element;
```

Comment:
nit: can you update the jsdoc

(If you want, this extension is super helpful for jsdoc, I use it constantly: https://marketplace.visualstudio.com/items?itemName=oouo-diogo-perdigao.docthis)

Reply:
Fixed this and updated other components to use correct JSDoc formatting